### ISSUE TITLE
Numerictextbox TypeError: e.toFixed is not a function

### ISSUE THREAD

**Issue by ilianiv** [telerik/kendo-angular#1255]
## I'm submitting a...

* Bug report 

## Current behavior

Component throws a TypeError: e.toFixed is not a function when min/max input is bound to string and value is populated with min/max value through spinner buttons.

## Expected behavior

Cast to number or validate min/max inputs.

## Minimal reproduction of the problem with instructions

Plnkr: https://plnkr.co/edit/cysRyZCgbcPIY7BRJWic?p=preview

1. Click Up spinner button -> value becomes '2' as min value
2. Click Up spinner button again -> throws the error (think it tries to call '2'.toFixed())

## What is the motivation or use case for changing the behavior?
Some backend APIs return numbers as strings and you may bind such value to min or max input.

## Environment

Package versions:

```
@progress/kendo-angular-inputs@1.4.2
```

**Comment by ggkrustev** [comment]
Indeed, the component doesn't parse the input explicitly as it expects to receive Number value:

https://www.telerik.com/kendo-angular-ui-develop/components/inputs/api/NumericTextBoxComponent/#toc-min
**The required type is `number`.**

Unfortunately, the `min="2"` property definition will set the `min` value to string value.

The workaround in this case is just to use the `min` Input property instead:

```
<kendo-numerictextbox [value]="value" [min]="2">
</kendo-numerictextbox>
```

**Comment by ilianiv** [comment]
Yes, it's easy in this simple example. But in my project I have 100+ kendo-numerictextbox component and some of them are bound to variables that came from API calls and it's hard to figure out which component's min/max is bound to string. So I hope you'll implement the explicit parse to Number 🙏

**Comment by ipeshev** [comment]
Fixed in v2.1.0-dev.201802021248
